config/default/smtp.settings.yml

```diff
@@ -0,0 +1,21 @@
+_core:
+  default_config_hash: HENvUIeX6xNPGRB1w3z6rT9a71LM_hKXATF6XrM9WvM
+smtp_on: false
+smtp_host: smtp.gmail.com
+smtp_hostbackup: ''
+smtp_port: '465'
+smtp_protocol: ssl
+smtp_autotls: true
+smtp_timeout: 30
+smtp_username: quiltme@fourkitchens.com
+smtp_password: ''
```

Comment:
Not setting the password here to keep this a bit more secure. Will add this config to the live environment and then set it to be ignored by configuration changes.